roboteam_ai/src/skills/CoachDefend.cpp

```diff
@@ -20,6 +20,10 @@ bt::Node::Status CoachDefend::onUpdate() {
     if (! targetLocation) {
 //        std::cerr << "Could not find the location of defender " << robot->id << " in calculated positions!"
 //                  << std::endl;
+        command.x_vel = 0;
```

Comment:
looks good, after practice tournament this might be improved to finding a 'default good position' or something

Reply:
Yes.